db-service/test/cqn2sql/__snapshots__/delete.test.js.snap

```diff
@@ -1,8 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`delete test complex cascade delete for entity with 'not exists' 1`] = `"DELETE FROM Books as Books WHERE ( Books.author_id is not null or Books.author_version is not null ) and not exists (SELECT 1 as _exists FROM Author as Author left JOIN Books as parent ON parent.ID = Author.parent_ID WHERE Author.parent_ID = parent.code and parent.descr = Author.version)"`;
+exports[`delete test complex cascade delete for entity with 'not exists' 1`] = `"DELETE FROM Books as Books WHERE ( Books.author_id is not null or Books.author_version is not null ) and not exists (SELECT ? as _exists FROM Author as Author left JOIN Books as parent ON parent.ID = Author.parent_ID WHERE Author.parent_ID = parent.code and parent.descr = Author.version)"`;
```

Comment:
Is there a way to detect SELECT 1? That's a common pattern in our implementation that we could also try to put in SQL string.

Reply:
This is also a requirement for it to work with the `hdb` driver on HANA. As it checks the datatype of the place holder and of the data send to the placeholder. When doing `SELECT ?` the type of `?` resolves to `VARCHAR` and then sending `1` the driver rejects the request. So now when a column is a `{val:...}` it will always be inlined as otherwise all type information is lost.